.snippets/text/xcm/register-local-xc-20s.md

```diff
@@ -0,0 +1,74 @@
+The multilocation will include the parachain ID of Moonbeam, the pallet instance, and the address of the ERC-20. The pallet instance corresponds to the index of the ERC-20 XCM Bridge Pallet, as this is the pallet that enables any ERC-20 to be transferred via XCM.
+
+**Local XC-20s that are registered on other chains must comply with the standard ERC-20 interface as described in [EIP-20](https://eips.ethereum.org/EIPS/eip-20){target=_blank}.**
```

Comment:
I don't think this is strictly necessary, why would you say this?

Reply:
This was just copy paste from the other file. I'm pretty sure you had said that it needed to strictly follow the standard ERC-20 and it could't be customized

Reply:
Ah ok I understand the paragraph now, I was thinking that it stated that the asset registered on other chains must comply with the ERC-20 standard (like in that other chain).

Maybe something like:

"ERC-20s that aim to become XC-20s, that is, going to be registered on other chains, must comply with the..... on Moonbeam"?

Or something like that. I think ERC-20s become XC-20s once other parachains registers them